### DISCUSSION TITLE
Quarkus support?

### DISCUSSION THREAD

**Discussion by tigerinus** [maciejwalkowiak/just#36] [Ideas]
Great tool for Spring Boot developers.

Any thoughts on supporting Quarkus, and/or other frameworks?

**Comment by maciejwalkowiak** [comment]
I have it in mind but considering Quarkus has [Dev Mode](https://quarkus.io/guides/maven-tooling#dev-mode) and [Dev Services](https://quarkus.io/guides/dev-services) I am not sure what gap could Just fill. Do you have some ideas?

**Reply by juniormichieletto** [reply]
Nice tool Maciej :)
It could be beneficial to use the same command to run all projects,
also, the automatic docker-compose up that JUST could be nice for Quarkus Projects as well :)

When working with a set of services that include both Spring Boot and Quarkus,
it can become challenging to manage the different build tools and commands (especially for beginners or new joiners).
For example, when dealing with three services:
one Spring Boot with Maven, one Spring Boot with Gradle, and one Quarkus with Gradle, you would need to run three separate commands such as:

```bash
mvn spring:run
gradle bootRun
gradle quarkusDev
```

It could be nice to improve the developer experience :)

What do you think?

**Reply by maciejwalkowiak** [reply]
It makes sense. Thanks for a suggestion!

**Reply by juniormichieletto** [reply]
Welcome 😄  
btw, if you plan to add Quarkus compatibility I can try help develop or test
Thanks
